ts/Data/DataTable.ts

```diff
@@ -681,6 +697,7 @@ class DataTable implements DataEvent.Emitter {
 
         const column = table.columns[columnNameOrAlias];
 
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
What is the reason for it? I saw above that there is a problem with the linter, but what kind of issue?

Reply:
The issue is described [here](https://typescript-eslint.io/rules/restrict-template-expressions/).